Makefile

```diff
@@ -98,7 +98,7 @@ $(ELF): $(OBJS) $(LDSCRIPT)
 # Build c sources, and ensure alignment
 $(C_BUILDDIR)/%.o : $(C_SUBDIR)/%.c
 	$(CPP) $(CPPFLAGS) $< | $(CC1) $(CC1FLAGS) -o $(C_BUILDDIR)/$*.s
-	@echo ".text\n\t.align\t2, 0\n" >> $(C_BUILDDIR)/$*.s
```

Comment:
Is this a linux issue? This breaks the build on MacOS

Reply:
The original one won't translate \n and \t into newline and tab in linux. I actually checked pokeemerald and they also used `-e` :/